tests/test_utils.py

```diff
@@ -15,7 +15,7 @@
 from unittest import mock
 
 from commoncode.testcase import FileDrivenTesting
-from test_cli import check_json_file_results
+from .test_cli import check_json_file_results
```

Comment:
This is not a module, so why this code change? Also do not make test code changes unrelated to the rest.

Reply:
Same reason as stated above. pytest fail to import it,  PEP recommendation to use relative imports, etc..
Is really not unrelated if you want to make tests pass, or mess with PYTHONPATH. Here's what's happens:

![image](https://github.com/user-attachments/assets/c3aba17b-2962-44c4-8d6a-b723cf6554af)

Reply:
Removed init files and used pythonpath direct on pytest